### ISSUE TITLE
[Question] How is battlesnake.io related to https://github.com/Dkendal/battle_snake ?

### ISSUE THREAD

**Issue by pydo** [BattlesnakeOfficial/engine#4]
I found this project  https://github.com/Dkendal/battle_snake when searching for battle snake bots. 
It looks like `battlesnake.io` engine was based off the project. If that's the case, is battlesnake.io some form of spiritual successor or was it an attempt to re-implement the original game since it's no longer maintained?

**Comment by pydo** [comment]
The [about](https://play.battlesnake.com/about/mission/) and [FAQ](https://play.battlesnake.com/faq/) (404) don't mention anything about https://github.com/Dkendal/battle_snake

**Comment by bvanvugt** [comment]
@pydo thanks for the question! We've been running Battlesnake since 2015, and the game engine has gone through a few iterations since then (mostly due to scaling requirements). 

The repo you mention is an older, sponsor-built version of the Battlesnake game engine, written in 2017 by Dkendal and the Stembolt team for an event they sponsored in Victoria that year. We decided to go in another direction for the official engine, and that experimental version remains open source.

As battlesnake.com grows we've been able to move away from third-party and community code, and all official development now happens in this GitHub organization. As you suggest, it might be interesting to add notes about past engines to docs/faqs :) Possibly we could open source more of the older engines too.
